extensions/objects/templates/literally-canvas-diagram-editor.component.spec.ts

```diff
@@ -18,16 +18,24 @@
 
 describe('LiterallyCanvasDiagramEditor', function() {
   var LCDiagramEditorCtrl = null;
+  var mockLiterallyCanvas = {
+    setImageSize: function(width, height) {
+      var text = 'The updated diagram width is ' + width +
```

Comment:
Same comments as before re using "(" and indentation. (Please follow them for future PRs.)

Reply:
Done.